src/qt/res/icons/x.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Perhaps we should give the file the same name as the alias. Took me a second to figure out what this is used for.

Reply:
Done.